pkg/util/workloadspread/workloadspread.go

```diff
@@ -791,11 +800,13 @@ func initializeWorkloadsInWhiteList(c client.Client) {
 			})
 		}
 	}
+	klog.InfoS("initialized workload list", "workloads", workloads)
```

Comment:
and , "workloadSpread", klog.KObj(ws)

Reply:
The effect of this function is global and it will execute only once.